common/biovotion/biovotion_vsm_heart_rate.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.biovotion",
+  "type": "record",
+  "name": "BiovotionVSMHeartRate",
+  "doc": "Heart rate calculated by biovotion device.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "HeartRateValue", "type": "float", "doc": "Heart rate value (bpm)"},
+    {"name": "HeartRateQuality", "type": "float", "doc": "Heart rate quality (0-1)"}
```

Comment:
perhaps lower camel case? A reference to what the quality means would be helpful